frontend/src/pages/RegisterPage.jsx

```diff
@@ -1,7 +1,23 @@
+const LABEL = "block text-dark/50 font-semibold text-2xl"
+const INPUT_FIELD = "rounded-md pl-3 h-14 w-full border-solid border-2 border-dark/50 mb-4 text-lg"
+//Need to add states for inputs
 function RegisterPage() {
 	return ( 
-		<div className="bg-dark min-h-screen pt-36 text-white">
-			<h1 className="pt-36">RegisterPage</h1>
+		<div className="bg-dark min-h-screen pt-36 text-dark w-screen flex flex-col items-center">
+            <h1 className="w-[500px] font-semibold text-3xl mb-2 text-white">REGISTER</h1>
```

Comment:
Ik in the Figma I had the text in the middle but I think it look better left aligned so try to make this left aligned